lib/cdo/contact_rollups.rb

```diff
@@ -53,6 +53,21 @@ class ContactRollups
   # PD courses in quoted, comma-separated form for inclusion in SQL IN clauses
   COURSE_LIST = COURSE_ARRAY.map {|x| "'#{x}'"}.join(',')
 
+  CSF_SCRIPT_ARRAY = [
+      "course1",
+      "course2",
+      "course3",
+      "course4",
+      "coursea",
+      "courseb",
+      "coursec",
+      "coursed",
+      "coursee",
+      "coursef"
+  ].freeze
```

Comment:
Is this not defined anywhere else? Should it be? Can we retrieve the list of script names from somewhere else so it's up to date (in case it changes)? It does appear to also be hardcoded on https://code.org/educate/curriculum/elementary-school ([source](https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/code.org/public/educate/curriculum/elementary-school.md#courses-a-f)), so maybe not :(

Reply:
I think the right thing to do is to make an entry in the `course` table for csf and designate these scripts as belonging to it via `course_scripts` as we do for CSD & CSP. Not sure why we don't and there appears to be some history here. Am pursuing that separately.